Tukui -- Failed to find addon version number for game version

**Describe the bug**
Tukui -- Failed to find addon version number for game version:

**If you run `pipenv update` on the command line and try again, does the issue go away?**
No

**Log output**
    2020-12-04 03:30:07,394 [DEBUG] [19068] [urllib3.connectionpool] https://www.tukui.org:443 "GET /addons.php?id=186 HTTP/1.1" 200 7291
    2020-12-04 03:30:07,396 [ERROR] [19068] [updater.manager.addon_manager] Failed to find addon version number for game version: GameVersion.agnostic, https://www.tukui.org/addons.php?id=186
    Traceback (most recent call last):
      File "D:\Games\WoWAddonUpdater\updater\site\tukui.py", line 55, in get_latest_version
        version = re.search(f'>{self._version_pattern}<', text).group(1)
    AttributeError: 'NoneType' object has no attribute 'group'

    The above exception was the direct cause of the following exception:

    Traceback (most recent call last):
      File "D:\Games\WoWAddonUpdater\updater\manager\addon_manager.py", line 100, in update_addon
        latest_version = site.get_latest_version()
      File "D:\Games\WoWAddonUpdater\updater\site\tukui.py", line 59, in get_latest_version
        raise self.version_error() from e
    updater.site.abstract_site.SiteError: Failed to find addon version number for game version: GameVersion.agnostic, https://www.tukui.org/addons.php?id=186

--- a/.gitignore
+++ b/.gitignore
@@ -30,3 +30,5 @@ dist/
 wow_addon_updater.egg-info/
 
 test-addon-directory/
+
+AddOns - Shortcut.lnk

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,7 +1,7 @@
 # Changelog
 
 ## Unreleased
-- 
+- Fix the tukui regex that finds the version number to install and record.
 
 ## v1.8.0 - 09/09/2020
 - Add support for installing as a module and have a command named `wow-addon-updater` available. (Thanks @elebertus)

--- a/test/site/test_tukui.py
+++ b/test/site/test_tukui.py
@@ -7,7 +7,8 @@ version_test_data = [
     ['https://www.tukui.org/classic-addons.php?id=2', 'https://www.tukui.org/classic-addons.php?download=2', 'ElvUI'],
     ['https://www.tukui.org/download.php?ui=tukui', 'https://www.tukui.org/downloads/tukui-', 'Tukui'],
     ['https://www.tukui.org/download.php?ui=elvui', 'https://www.tukui.org/downloads/elvui-', 'ElvUI'],
-    ['https://www.tukui.org/addons.php?id=3', 'https://www.tukui.org/addons.php?download=3', 'AddOnSkins']
+    ['https://www.tukui.org/addons.php?id=3', 'https://www.tukui.org/addons.php?download=3', 'AddOnSkins'],
+    ['https://www.tukui.org/addons.php?id=186', 'https://www.tukui.org/addons.php?download=186', 'CopyPasta2']
 ]
 
 

--- a/updater/site/tukui.py
+++ b/updater/site/tukui.py
@@ -18,7 +18,7 @@ class Tukui(AbstractSite):
 
     _page: BeautifulSoup = None
 
-    _version_pattern = r'(?P<version>[\d]+\.[\d]+)'
+    _version_pattern = r'(?P<version>([\d]+\.)+[\d]+([\d]+)?)'
 
     def __init__(self, url: str):
         super().__init__(url, GameVersion.agnostic)
@@ -47,12 +47,12 @@ class Tukui(AbstractSite):
     def get_latest_version(self):
         try:
             if self._is_special_tukui_link():
-                version = re.search(self._version_pattern, self.find_zip_url()).group(1)
+                version = re.search(self._version_pattern, self.find_zip_url()).group('version')
             else:
                 response = Tukui.session.get(self.url + '#extras')
                 response.raise_for_status()
                 text = response.text
-                version = re.search(f'>{self._version_pattern}<', text).group(1)
+                version = re.search(f'>{self._version_pattern}<', text).group('version')
             self.latest_version = version
             return version
         except Exception as e:

